### DISCUSSION TITLE
Email Channel - departments status. (Open, closed, in progress etc.)

### DISCUSSION THREAD

**Discussion by wiredhosting** [chatwoot/chatwoot#9882] [Q&A]
Hi, how are you doing folks? I am testing the self hosted version of ChatWoot and I wanted to know if I can add different status for each department.

For example

Support
 - Open
 - In progress
 - Closed

Sales
- Open
- Fraud Review
- Awaiting payment
- Shipped

Thanks.

**Comment by iammattmartin** [comment]
This is not yet possible.
